addons/website/views/website_templates.xml

```diff
@@ -1837,23 +1838,18 @@
                         <li class="list-inline-item mx-3"><i class="fa fa-1x fa-fw fa-phone me-2"/><span class="o_force_ltr"><a href="tel:+1 555-555-5556">+1 555-555-5556</a></span></li>
                         <li class="list-inline-item mx-3"><i class="fa fa-1x fa-fw fa-envelope me-2"/><span><a href="mailto:info@yourcompany.example.com">info@yourcompany.example.com</a></span></li>
                     </ul>
-                </div>
-            </section>
-            <section class="s_text_block pt16 pb16" data-snippet="s_text_block" data-name="Logo">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-lg-2 offset-lg-5">
+                    <div class="col-lg-2 offset-lg-5">
```

Comment:
It's definitely not valid bootstrap to have a `ul` followed by a column. Columns are meant to be direct children of rows, Make this column fullwidth and you'll probably have a double padding at the start for example.